src/components/ReviewList.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import ReviewCard from "./ReviewCard";
+import Loading from "./Loading";
+
+const ReviewList = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetch("https://northcoders-games-project.herokuapp.com/api/reviews")
+      .then((response) => response.json())
+      .then(({ reviews }) => {
+        setReviews(reviews);
+        setIsLoading(false);
+      });
+  }, []);
+
+  return (
+    <section>
+      <Loading isLoading={isLoading}>
+        <ul className="review-list">
+          {reviews.map((review) => {
+            return (
+              <ReviewCard
```

Comment:
As stated before, it's probably neater to send through the whole review object here and then destructure it in the ReviewCard component. This saves having to do it twice!